packages/utilities/psammead-locales/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.10.0 | [PR#2176](https://github.com/bbc/psammead/pull/2176)  Add `es` (Spanish) locale and update `ar` (Arabic) and `pt-br` (Brasil) locales |
```

Comment:
Can you please also update the [Readme.md](https://github.com/bbc/psammead/tree/a81b812f2b7eb49bd5f33ac8642e32433ea3e186/packages/utilities/psammead-locales#moment-locales) to include the `es` locale?

Reply:
Sure, I missed it.